File: PatientFinal/PatientFinal/DTOs/CityDto.cs
namespace PatientFinal.DTOs;

public class CityDto
{
    public string name { get; set; }
    public string code { get; set; }
    
    public CityDto() {}
    
    public CityDto(string name, string code) {

        this.name = name;
        this.code = code;
    }
}

File: PatientsServer/src/main/java/DTOs/CityDto.java
package DTOs;

public class CityDto
{
  private String name;
  private String code;


  public CityDto(String name, String code) {
    this.name = name;
    this.code = code;
  }


  public String getName() {
    return name;
  }

  public String getCode()
  {
    return code;
  }

  public void setCode(String code)
  {
    this.code = code;
  }

  public void setName(String name)
  {
    this.name = name;
  }
}
